v2/apiserver/internal/api/secrets_test.go

```diff
@@ -197,7 +197,7 @@ func TestSecretsServiceSet(t *testing.T) {
 		{
 			name: "unauthorized",
```

Comment:
If you could possibly change the name of this test case to "user does not have read permissions" it would be more clear and then everything else looks great!

Reply:
Yeah agree.